modules/keyboardshortcut/keyboardshortcut.js

```diff
@@ -139,6 +139,9 @@ var KeyboardShortcut = {
      * @returns {string} e.key or something close if not supported
      */
     _getKeyboardKey: function (e) {
+        if (e.which === 32) {
```

Comment:
Event object contains "key" property when SPACE is hitted so we've got early return from "_getKeyboardKey" and "switch" is not helpful. I've proposed another approach.